dashboard-for-web/articles/web-dashboard-designer-mode/appearance-customization/conditional-formatting.md

```diff
@@ -27,68 +27,87 @@ Comparison rules used in conditional formatting can be divided into the followin
 * **Bar Color Ranges** - Allows you to visualize numeric values using bars whose colors are contained in the specified color set.
```

Comment:
I'd like to clarify this definition. My understanding is that each color represents a range of values. If the value falls within a specific range, the color specified by that range is displayed. Is this correct?
If so, can we say:

Bar Color Ranges - Visualizes numeric values as bars. Values within a range display a specific color.

Reply:
```suggestion
* **Bar Color Ranges** - Visualizes numeric values as bars. Values within a range display a specific color.
```

Reply:
Please change to:

* **Bar Color Ranges** - Visualize numeric values as bars. Values within a range display a specific color.